### ISSUE TITLE
Consider allowing `daptm:scriptType` on Script Events

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/dapt#140]
At the moment we only define a semantic for `daptm:scriptType` on the `tt` element, i.e. at the DAPT Script level. Should we also define what it means if someone applies it at the Script Event level?

I could imagine, related to #52, a script document that has:

Script level `tt@daptm:scriptType = "originalTranscript"` containing:
* Script Event level `div@daptm:scriptType="translatedTranscript"`
* Script Event level `div@daptm:scriptType="translatedTranscript"`
* Script Event level `div@daptm:scriptType="originalTranscript"`

This could be used to signify that two out of the three script events have been translated, but the third has not.

If we were adopting this then we could specify min/max rules about the permitted value of `tt@daptm:scriptType` for a given set of `div@daptm:scriptType` values present within the document. In the above example, we might specify that the "lowest" value must be specified at the script level, so if any originalTranscript script events are present, then the script level script type must be originalTranscript.

An alternative option would be to prohibit `daptm:scriptType` on any element other than `tt`.

I suspect that if we leave this unspecified then some implementers will do something non-standard with it, which may or may not be what others would see as useful.

**Comment by cconcolato** [comment]
> This could be used to signify that two out of the three script events have been translated, but the third has not.

Can't this be detected by comparing the computed values of lang and langSrc?

**Comment by nigelmegitt** [comment]
> Can't this be detected by comparing the computed values of lang and langSrc?

It would have to be more complex than that because there could be some original language content that happens to be in the target language for the translation, so there would be multiple variables to compare. In the end I suspect it might need to be a human decision whether to label the script event as originalTranscript or translatedTranscript.

**Comment by cconcolato** [comment]
> It would have to be more complex than that because there could be some original language content that happens to be in the target language for the translation

I'm missing your point. In that case they have not been translated.

**Comment by nigelmegitt** [comment]
> > It would have to be more complex than that because there could be some original language content that happens to be in the target language for the translation
> 
> I'm missing your point. In that case they have not been translated.

No, that's sort of my point (which I'm making badly) - there may be script events that are already in the target language and do not need translating, but that's something that a person would discover by going through the script events. Labelling each one with the target script type for the current phase of work (e.g. translation) as it is processed allows the remaining volume of work to be assessed or computed.

A single script event of this type (i.e. already in the target language) could reasonably be either labelled as either `originalTranscript` _or_ `translatedTranscript` without anything else changing.
